src/maintainer/knowledge_base.rst

```diff
@@ -554,45 +554,96 @@ Special packages
+You should specify only the libraries the package needs. (i.e. if the package
+doesn't need LAPACK, remove liblapack and liblapacke)
+
+At recipe build time, above requirements would download the NETLIB's reference
+implementations and build your recipe against those.
+At runtime by default the following packages will be used,
+
+.. code-block:: yaml
+    - openblas   # [not win]
+    - mkl        # [win]
+
+If a package needs a specific implementation's internal API for more control you can have,
+
+.. code-block:: yaml
+    requirements:
+      host:
+        - {{ blas_impl }}
+      run:
+        - libblas * *{{ blas_impl }}
+        - {{ blas_impl }}
+
+This would give you a matrix builds for different blas implementations. If you only want to support
+a specific blas implementation,
+
+.. code-block:: yaml
+    requirements:
+      host:
+        - openblas
+      run:
+        - libblas * *openblas
+        - openblas
+
+.. note::
+`blas_*` features should not be used anymore.
+
+Switching BLAS implementation
+"""""""""""""""""""""""""""""
+
+You can switch your BLAS implementation by doing,
 
-  The information regarding the blas package may now be outdated and needs checking.
+.. code-block:: bash
+    conda install "libblas=*=*mkl"
+    conda install "libblas=*=*openblas"
+    conda install "libblas=*=*blis"
+    conda install "libblas=*=*netlib"
+
+This would change the BLAS implementation without changing the conda packages depending
+on BLAS.
 
-BLAS metapackage
-................
+Following legacy commands are also supported as well.
 
-   -  Version will have two values X.Y
+.. code-block:: bash
+    conda install "blas=*=mkl"
+    conda install "blas=*=openblas"
+    conda install "blas=*=blis"
+    conda install "blas=*=netlib"
 
-      -  X represents changes to the metapackage.
-      -  Y represents priority of BLAS (if we change priorities BLASes X
-         must be bumped, if we want to prioritize something new over
-         OpenBLAS we do not need to change X)
+How it works
+""""""""""""
 
-         -  1 is OpenBLAS
-         -  0 is None (no BLAS whatsoever)
+At recipe build time, the netlib packages are used. This means that the downstream package will
+link to ``libblas.so.3`` in the ``libblis=*=*netlib` and will use only the reference
+implementation's symbols.
 
-   -  needs to have version stay the same across all variants.
-   -  build number cannot be touched (it won't be in the string anyways)
-   -  no pinning of BLAS inside the metapackages (dependencies can pin
-      this)
-   -  to preserve a BLAS in an environment it is recommend to add
-      ``pinned`` to ``conda-meta`` and specify down to the build string
-      what is the expected BLAS
-   -  To install a specific variant, ``conda install blas=1.0=none`` /
-      ``conda install blas=1.0=openblas``. It is hoped the syntax will be
-      improved in conda.
+``libblas`` and `libcblas`` versioning is based on the Reference LAPACK versioning which at the
```

Comment:
missed a backtick for `libcblas` here and two lines hence

Reply:
Thanks